internal/rundata/dut.go

```diff
@@ -81,8 +81,13 @@ func (di *dutInfo) setFromComponentChassis(ctx context.Context, y components.Y)
 	}
 
 	if desc := component.Description; desc != nil {
-		if di.model == "" || strings.HasPrefix(*desc, "Cisco ") {
-			di.model = *component.Description
+		switch {
+		case strings.HasPrefix(di.vendor, "Cisco"):
+			fallthrough
+		case strings.HasPrefix(di.vendor, "Nokia"):
```

Comment:
Per the sample output (https://github.com/openconfig/featureprofiles/pull/899#issuecomment-1349986107), `Description` does not contain Nokia, instead it has the actually chassis model name?

Reply:
The "Nokia" prefix is checked on di.vendor which comes from mfg-name, not the description.